docker/substra-tests-stack/bin/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+set -v
+
+DEFAULT_REPO="$1"
+SUBSTRA_TESTS_BRANCH="$2"
+
+# Init helm
+helm init --upgrade --force-upgrade --wait
+
+# Clone repos
+rm -rf src
+mkdir -p src
+cd src
+git clone --depth 1 https://github.com/SubstraFoundation/substra-backend.git
+git clone --depth 1 https://github.com/SubstraFoundation/hlf-k8s.git
+git clone --depth 1 https://github.com/SubstraFoundation/substra-tests.git --branch "${SUBSTRA_TESTS_BRANCH}"
+cd ..
+
+# Patch skaffold files for kaniko
+python -m venv .venv
+source .venv/bin/activate
+pip install pyyaml
```

Comment:
Why are you running the pip install command at runtime instead of running it in your Dockerfile ?
You could use something like this to install directly in your virtual environment: https://pythonspeed.com/articles/activate-virtualenv-dockerfile/

Reply:
We can install pyyaml directly within the Dockerfile. No need to use a virtualenv in docker.
It's surely a remaining from previous script setup